package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@quickcase/node-logging",
-  "version": "3.0.3",
+  "version": "4.0.0",
```

Comment:
Version should not be changed manually, but instead using `npm version major` to ensure correct alignment of `package-lock.json` and consistent tagging. Bumping version should ideally be performed on the `main` branch so that the resulting tag is visible on the main trunk.

Please revert the version for now and we'll bump it directly on `main` after the PR is merged

Reply:
reverted